devices/eurotronic/eurotronic_spzb0001_trv.json

```diff
@@ -0,0 +1,373 @@
+        "$address.ext",
+        "0x01",
+        "0x0201"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0301",
+        "endpoint": "0x01",
+        "in": [
+          "0x0000",
+          "0x0001",
+          "0x0201"
+        ]
+      },
+      "meta": {
+        "values": {
+          "config/mode": {
+            "off": 0,
+            "auto": 1,
+            "heat": 4
+          }
+        }
+      },
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/battery",
+          "refresh.interval": 43265,
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "eval": "Item.val = Attr.val / 2",
+            "fn": "zcl:attr"
+          },
+          "read": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "config/displayflipped",
+          "refresh.interval": 3660,
+          "read": {
+            "at": "0x4008",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "write": {
+            "at": "0x4008",
+            "cl": "0x0201",
+            "dt": "0x22",
+            "ep": 1,
+            "eval": "if (Item.val == true) { R.item('config/hostflags').val | 0x02 } else { R.item('config/hostflags').val & ~0x02}",
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "parse": {
+            "at": "0x4008",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "Item.val = ((Attr.val & 0x02) !== 0);",
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "default": false
+        },
+        {
+          "name": "config/heatsetpoint",
+          "refresh.interval": 660,
+          "read": {
+            "at": "0x0012",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "write": {
+            "at": "0x0012",
+            "cl": "0x0201",
+            "dt": "0x29",
+            "ep": 1,
+            "eval": "Item.val",
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0012",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
```

Comment:
This one here is very delicate, as the screwed firmware version does not allow for any differentiation which attribute must be **written**. For the Spirit, the legacy code uses mf attribute 0x4003, so I'd vouch for continuing using it in the _read_ and _parse_ functions.

As for the write function, I guess we can leave it as is, since it doesn't matter anyway. The legacy code has an ugly, but required workaround and sends a write request to any relevant attribute (so in total 2). As said, this must remain as you have no chance to know which attribute the hardware really accepts.

Please, this must be tested in strict mode!